tests/cpp/fwklib/RegionHelper.hpp

```diff
@@ -85,10 +83,25 @@ class RegionHelper {
     return sString;
   }
 
+  static std::string to_string(ExpirationAction expirationAction) {
```

Comment:
Yes! I'm glad to see this approach to `to_string(...)` instead of putting it on the class itself.